Podfile

```diff
@@ -14,7 +16,7 @@ target 'Video-Internal' do
   pod 'FirebaseUI/Google', '~> 9'
   pod 'IGListDiffKit', '~> 4'
   pod 'KeychainAccess', '~> 4'
-  pod 'TwilioVideo', '~> 3'
+  pod 'TwilioVideo', '~> 4.0.0-rc15'
```

Comment:
This will need to be changed to `4` before merge.

Reply:
Will approve but yes, please be sure to change this before merging.